Makefile

```diff
@@ -235,9 +235,7 @@ test: # Run unit tests on the REANA package.
 	pydocstyle reana
 	black --check .
 	check-manifest --ignore ".travis-*"
-	sphinx-build -qnNW docs docs/_build/html
 	python setup.py test
-	sphinx-build -qnNW -b doctest docs docs/_build/doctest
```

Comment:
Is removing Sphinx alltogether a good thing?  In this way `reana.rtfd.io` won't be updated... I guess we should keep a tiny sphinx page with ultra little content, e.g. just advertising `docs.reana.io` there, or perhaps redirecting automatically. Or can it be done manually on the RTFD dashboard console? (in which case we could kill sphinx docs indeed).

Reply:
`index.rst` restored.